python/nammu/view/SyntaxHighlighter.py

```diff
@@ -168,8 +168,8 @@ def syntax_highlight(self, top_line=None, bottom_line=None,
 
         # when we have arabic text, we need to fix an off by 1 error caused
```

Comment:
I honestly don't understand what this comment refers to.  `self.styledoc.getLength()` is equal to `self.controller.controller.arabicIndex` in a newly opened file.  However, if I subtract 1 to `self.styledoc.getLength()` I have to add an extra character (e.g., a space) in order to get syntax highlighting on a word.  E.g., in the recipe in #393, I have to write "`@obverse `" (note the extra space after "`@obverse`") in order to highlight the keyword if I subtract 1.  I couldn't reproduce any off-by-1 error while testing this PR locally.

Reply:
I'm also trying to make sense of it and failing. Your solutions seems sensible so we can ignore this comment. Please delete so it doesn't cause more confusion in the future.